app/models/conference.rb

```diff
@@ -22,7 +22,7 @@ class Conference < ActiveRecord::Base
   has_many :supporters, through: :ticket_purchases, source: :user
   has_many :tickets, dependent: :destroy
   has_many :resources, dependent: :destroy
-
+  has_many :votable_fields, dependent: :destroy
```

Comment:
Then probably we should move VotableField to be a program option instead of conference in all places including views and routes of the app as well. I will be doing that as well. :)